test/record.js

```diff
@@ -0,0 +1,62 @@
+const nock = require('nock');
+const path = require('path');
+const fs = require('fs');
+const zlib = require('zlib');
+
+module.exports = function (name, options) {
+  // options tell us where to store our fixtures
+  options = options || {};
+  const fixtureFolder = options.fixtureFoldr || 'fixtures';
+  const fp = path.resolve(__dirname, path.join(fixtureFolder, name + '.js'));
+  // `hasFixture` indicates whether the test has fixtures we should read,
+  // or doesn't, so we should record and save them.
+  // the environment variable `NOCK_RECORD` can be used to force a new recording.
+  let hasFixture = process.env.NOCK_RECORD === 1;
```

Comment:
Based on your comment I'd expect it to detect if a fixture is present or not. But you just hardcoded it based on the environment variable. When I look at the npm scripts, it is always 0, so hasFixture would always be false, creating new fixtures every run. How does this work? 🤔

Reply:
Sometimes we want to refresh the fixtures if for example, we add a new endpoint so we change `NOCK_RECORD` to `1`